_posts/2017-10-30-lambda-world-conference.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+asset-type: post
+name: Lamda-World-Conference-2017
+title: 'Lambda World Conference 2017'
+date: 2017-10-30 12:00:00 +00:00
+author: Raquel M Carmena
+image:
+  src: /assets/img/custom/blog/2017-10-30-lambda-world-conference.jpg
+tags:
+- functional programming
+- category theory
+- lambda calculus
+- conference
+
+---
+Thanks Codurance for giving me the opportunity of attending <a href="http://www.lambda.world/" target="_blank">**Lambda World Conference 2017**</a> last 26th and 27th October in Cádiz, the largest _Functional Programming_ event in Spain consisting of two intense days:
+
+* A half-day of open space 
+* A half-day of workshops
+* A full day of talks
+
+**Rúnar Bjarnason**, one of the authors of the _red book_ of Scala, improvised a crash course about Category Theory during the open space and it was very absorbing how he identified objects and arrows in every category.
```

Comment:
Of the Scala red book sounds better I think

Reply:
OK! Thanks!